third_party/move/move-model/src/builder/exp_builder.rs

```diff
@@ -2419,11 +2441,13 @@ impl<'env, 'translator, 'module_translator> ExpTranslator<'env, 'translator, 'mo
                     Operation::Select(mid, sid, FieldId::new(field_name))
                 } else {
                     // Create a placeholder for later resolution.
-                    self.placeholder_map
-                        .insert(id, ExpPlaceholder::FieldSelectInfo {
+                    self.placeholder_map.insert(
```

Comment:
Not sure if this is worth investigating, but this file seems to have a number of purely formatting related changes (like this one).

Reply:
I see what you mean, but `rustfmt +nightly` seems to be fine with it.  I think that changing `loc` to `_loc` in a few places put the line length over some limit.